code/modules/projectiles/projectile/special.dm

```diff
@@ -69,6 +69,7 @@
 	desc = "Boom"
 	icon_state = "missile"
 	damage = 30
+	ricochets_max = 0 //it's a MISSILE
```

Comment:
I'm not able to see it, but I might just be dumb.
https://github.com/tgstation/tgstation/blob/master/code/modules/projectiles/projectile.dm#L140